conans/test/functional/generators/assets/cmake_find_package_multi/project_global/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(PackageTest CXX)
+
+if(CONAN_LINK_RUNTIME)
+    foreach(flag CMAKE_C_FLAGS_RELEASE CMAKE_CXX_FLAGS_RELEASE
+                 CMAKE_C_FLAGS_RELWITHDEBINFO CMAKE_CXX_FLAGS_RELWITHDEBINFO
+                 CMAKE_C_FLAGS_MINSIZEREL CMAKE_CXX_FLAGS_MINSIZEREL)
+        if(DEFINED ${flag})
+            string(REPLACE "/MD" ${CONAN_LINK_RUNTIME} ${flag} "${${flag}}")
+        endif()
+    endforeach()
+    foreach(flag CMAKE_C_FLAGS_DEBUG CMAKE_CXX_FLAGS_DEBUG)
+        if(DEFINED ${flag})
+            string(REPLACE "/MDd" ${CONAN_LINK_RUNTIME} ${flag} "${${flag}}")
+        endif()
+    endforeach()
+endif()
+
+add_executable(example example.cpp)
+
+set(CMAKE_MODULE_PATH ${CMAKE_BINARY_DIR})
+find_package(bye)
+
+# Global approach
+if(bye_FOUND)
+   include_directories(${bye_INCLUDE_DIRS})
+   target_link_libraries(example ${bye_LIBRARIES})
+endif()
+
+if(hello_FOUND)
```

Comment:
Is it possible that bye_FOUND and not hello_FOUND? Shouldn't the find form bye return None if the transitive dependencies are not found?

Why it doesn't return ALL the includes and libraries to link, in one single variable? Seems cumbersome to use, specially if you have many dependencies.